types/levelup/index.d.ts

```diff
@@ -1,7 +1,8 @@
-// Type definitions for levelup 3.1
+// Type definitions for levelup 4.3
```

Comment:
Was `clear` introduced in 4.x? Is there a chance that there may need to be future updates to the typings for the 3.x version of the package side-by-side with the 4.x typings? If so you may need to first copy the 3.x typings into a `v3` folder: https://github.com/DefinitelyTyped/DefinitelyTyped#if-a-library-is-updated-to-a-new-major-version-with-breaking-changes-how-should-i-update-its-type-declaration-package

If it seems unlikely that the 3.x typings will need future revisions then let me know and I can merge this as-is.

Reply:
Looks like levelup has been on the 4.x major version since `Dec 23, 2018`...
But `clear` was just [added back in September](https://github.com/Level/levelup/commit/45e73a9fdd11062f358a8818738fd62b7f0e68fc).
It _seems_ unlikely that there would be changes to 3.x types to me... particularly if these typings are that far behind anyway?

Reply:
Just to follow up here, I think I'm saying it seems unlikely that the 3.x typings will need future revisions?